<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AttributeChangeDialog</class>
 <widget class="QDialog" name="AttributeChangeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Voce Computo - Associa etichette</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2">
    <widget class="QPushButton" name="setAttributesPushButton">
     <property name="text">
      <string>Imposta</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="addAttributesPushButton">
     <property name="text">
      <string>Aggiungi</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="removeAttributePushButton">
     <property name="text">
      <string>Rimuovi</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="cancelPushButton">
     <property name="text">
      <string>Annulla</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QTableView" name="attributesTableView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
